### ISSUE TITLE
Wykład 4 - patcher

### ISSUE THREAD

**Issue by wdebsqi** [daftcode/daftacademy-python_4beginners-autumn2020#13]
Przy pierwszym podejściu do zadania udawało mi się przejść wszystkie testy z wyjątkiem **mutable_elements**:
```
Traceback (most recent call last):
  File "/home/runner/unit_tests.py", line 64, in test_mutable_elements
    res_dict2 = deepcopy(super_dict)
NameError: name 'deepcopy' is not defined
```

Okazało się, że nie muszę zmieniać nic w logice programu, a jedynie dodać  `from copy import deepcopy`, żeby program przeszedł wszystkie testy.

Nie do końca rozumiem co tu się wydarzyło. `copy.deepcopy` jest wykorzystywane przy testach i stąd ten błąd? Jeśli tak to przydałoby się wspomnieć w treści zadania o konieczności zaimportowania tego modułu.

**Comment by JWilczynski97** [comment]
Czy wykorzystywałeś/aś funkcję deepcopy w swoim kodzie?

**Comment by wdebsqi** [comment]
Nie. Szczerze mówiąc nie wiedziałem nawet o jej istnieniu, bo w materiałach z wykładu nie była wspominana. Dopiero jak poszukałem w necie to pomyślałem, że być może jest używana w testach i dlatego ją zaimportowałem.

**Comment by JWilczynski97** [comment]
A to faktycznie ciekawe. Najwyraźniej w kodzie testów zabrakło tej linijki. Dobrze, że zwróciłeś uwagę - być może oszczędziłeś komuś kilku godzin zastanawiania się 😅

**Comment by ludaliaallen** [comment]
Dzięki, że piszesz, bo miałam ten sam problem :))
